content/analytics/index.md

```diff
@@ -8,7 +8,9 @@ title: "Analytics"
 type = 'card'
 header = '[devstats.scientific-python.org](https://devstats.scientific-python.org)'
 body = '''
-Generate developer statistics for a specified project using the [devstats](https://github.com/scientific-python/devstats) package.
+We generate developer statistics for various core packages using the [devstats](https://github.com/scientific-python/devstats) tool.
+[Data releases](https://github.com/scientific-python/devstats-data/releases) are made weekly.
```

Comment:
Is this in place already?

Reply:
It is! Due to rate limiting, the action could not complete for all core projects. I'm looking into parallelization to see if that helps.

Reply:
https://github.com/scientific-python/devstats-data/actions/runs/11867300849

Reply:
First build is live! https://github.com/scientific-python/devstats-data/releases